pkg/cli/login/loginoptions.go

```diff
@@ -336,17 +337,20 @@ func (o *LoginOptions) gatherAuthInfo() error {
 // prepareBuiltinExecPlugin sets up the ExecConfig correctly
 // with the given values
 func (o *LoginOptions) prepareBuiltinExecPlugin() (*kclientcmdapi.ExecConfig, error) {
-	issuerUrl, err := o.extractIssuerURLForOIDC()
-	if err != nil {
-		return nil, err
+	var err error
+	if o.OIDCIssuerURL == "" {
```

Comment:
IMO this must always be set

Reply:
If this is always be set, why are we extracting from oauth-authorization-server endpoint?

Reply:
From the oc point of view, this doesn't matter too much and if in the future, it is decided that users always have to set `issuer-url`, it is easy to add a validation. I think, we can move on as is.

Reply:
> If this is always be set, why/what are we extracting from oauth-authorization-server endpoint?
ideally we shouldn't extract anything from there